### PR TITLE
Here how I made simple version of my CV using HTML and CSS

### PR DESCRIPTION
**Pull request by VadymDemchenko**

### CODE REVIEW

File: HW25/homework_25.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+</head>
+<body>
+    <div class="header">
+        <div class="photo-container"><img class="photo" src="my_photo.jpg"></div>
+        <div class="name">Вадим Демченко</div>
+    </div>
+    <hr style="color:saddlebrown;">
+    <div class="container">
+        <div class="sidebar">
+            <div class="title" style="margin-top: 0px">
```
Comment (klymenok) [inline HW25/homework_25.html:16]: You can move `margin-top` to the css file

File: HW25/homework_25.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                    <div class="period">2019-2022</div>
+                    <div class="company">vivo Ukraine</div>
+                    <div class="position">Head of trade-marketing</div>
+                    <div class="responsibilities">• набір, навчання та супровід промоутерів на всіх етапах їх роботи
+                                              <br>• створення мотивації для промоутерів
+                                              <br>• установка цілей та контроль результатів промоутерів
+                                              <br>• відстеження показань ефективності, контроль виконання планів
+                                              <br>• відвідування магазинів з промоутерами, допомога у побудові лояльності з колективом</div>
+                </div>
+                <hr style="width: 98%">
```
Comment (klymenok) [inline HW25/homework_25.html:151]: Style can be moved to the css file

**Review by klymenok — APPROVED** [review]
Looks good! Check my comments

**Comment by VadymDemchenko** [comment]
Thanks for your comments

**Pull request merged**